CMakeLists.txt

```diff
@@ -73,6 +73,10 @@ option(WITH_CHISEL "Include chisel implementation" OFF)
 
 option(CREATE_TEST_TARGETS "Enable make-targets for unit testing" ON)
 
+if(CREATE_TEST_TARGETS AND NOT WIN32)
+	include(gtest)
+endif()
+
```

Comment:
gtest doesn't work on win32?

Reply:
I didn't try that, but it was not compiled on win32 before.